clouddriver-openstack/src/main/resources/asg.yaml

```diff
@@ -27,10 +27,46 @@ parameters:
     description: Load balancer to use for this group
   security_groups:
     type: comma_delimited_list
-    description: security groups to use
+    description: Security groups to use
   subnet_id:
     type: string
     description: Subnet used to allocate a fixed IP for each server
+  autoscaling_type:
+    type: string
+    description: Type of autoscaling to perform. can be cpu_util, network.incoming.bytes.rate, or network.outgoing.bytes.rate
+    default: cpu_util
+  scaleup_cooldown:
+    type: number
+    description: Minimum amount of time (in seconds) between scaleup operations
+    default: 60
+  scaleup_adjustment:
+    type: number
+    description: Amount by which to change the instance count. Can be positive or negative
```

Comment:
Well, probably doesn't make much sense. Scaleup should be positive, scaledown should be negative. Can't think of use cases to the contrary.